### ISSUE TITLE
How to import fonts from node_modules?

### ISSUE THREAD

**Issue by rwwagner90** [GrabarzUndPartner/nuxt-font-loader-strategy#152]
I've been trying a bunch of different `~`, `@/`, `node_modules`, `/node_modules` etc and I cannot seem to import fonts from node_modules. How do I do this?

**Comment by ThornWalli** [comment]
Do you have an example dependency?

**Comment by rwwagner90** [comment]
@ThornWalli we ended up needing:

```js
 importPathResolve: (file) => {
   return `~${file}`
 },
 ```

This allowed a src like `src: '@foo/core-styles/assets/fonts/Serif',` to work, where `@foo/core-styles` is a package in node_modules.

With that `importPathResolve` it fixed the path so it worked for all cases.

**Comment by ThornWalli** [comment]
I created a package with fonts and was able to integrate it without problems. (https://github.com/ThornWalli/test-font)

In that case no prefix is used.

![image](https://user-images.githubusercontent.com/8287751/106260051-46216000-6220-11eb-8963-27a1e26935db.png)

The files are available in the node_modules?

**Comment by rwwagner90** [comment]
@ThornWalli yes, `@foo/core-styles` is in node_modules. Perhaps the `@` is the problem in a scoped npm package name.

**Comment by ThornWalli** [comment]
@rwwagner90 I have applied the change for `importPathResolve`. Now all require calls are made by ~ .

So it should be finished?

**Comment by rwwagner90** [comment]
@ThornWalli I am not sure. If you want to release a new version, I can try it out.

**Comment by rwwagner90** [comment]
Looks like a new version was released. I will try it out soon.
